docs/user/datastores/index_basics.rst

```diff
@@ -64,6 +64,38 @@ To index an attribute, add an ``index`` key to the attribute descriptor user dat
 
 Setting the user data can be done in multiple ways. See :ref:`set_sft_options` for more details.
 
+Attribute indices also support a secondary, tiered index structure. This can improve attribute queries
+that also contain a spatial and/or temporal predicate. Unless configured differently, the default geometry
+and date attributes will be used to create a secondary Z3 of XZ3 index.
```

Comment:
Do you mean Z3 or XZ3?  

Also, referring to this a 'secondary' index is slightly confusing.

Reply:
Oh, I'm just confused.

Reply:
oh, yes it should be 'or', good catch.